src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -64,6 +75,16 @@ public function iAmLoggedInAs($email)
     }
 
     /**
+     * @Given /^I am logged in customer$/
+     */
+    public function iAmLoggedInCustomer()
+    {
+        $user = $this->securityService->logInDefaultUser($this->mink->getSession());
```

Comment:
`securityService` should not return `user`. What is more it should not create a new user (I suppose it is).